var/spack/repos/builtin/packages/pumi/package.py

```diff
@@ -41,6 +39,9 @@ class Pumi(CMakePackage):
             description="Enable Simmetrix SimModSuite Support: 'base' enables "
             "the minimum set of functionality, 'kernels' adds CAD kernel "
             "support to 'base', and 'full' enables all functionality.")
+    variant('simmodsuite_version_check', default=True,
+            description="Enable check of Simmetrix SimModSuite version."
+            "Disable the check for testing new versions.")
```

Comment:
This needs an extra space after the period or a shorter description.

Reply:
Thank you.  https://github.com/spack/spack/pull/14597/commits/d8446416ce66265ff7122a57ce3753302eb85be0